modules/light-clients/08-wasm/types/querier_test.go

```diff
@@ -107,3 +113,88 @@ func (suite *TypesTestSuite) TestCustomQuery() {
 		})
 	}
 }
+
+func (suite *TypesTestSuite) TestStargateQuery() {
+	typeURL := "/ibc.lightclients.wasm.v1.Query/Checksums"
```

Comment:
wonder if we should add any more test queries for something basic, query from x/bank or x/staking for example, this is probably fine tho!

Reply:
I will add more if others also want this